repos/system_upgrade/common/actors/checkselinux/tests/test_checkselinux.py

```diff
@@ -1,6 +1,11 @@
+import pytest
+
+from leapp import reporting
+from leapp.libraries.actor import checkselinux
+from leapp.libraries.common.testutils import create_report_mocked, CurrentActorMocked, produce_mocked
+from leapp.libraries.stdlib import api
+from leapp.models import Report, SELinuxFacts, SelinuxPermissiveDecision, SelinuxRelabelDecision
 from leapp.snactor.fixture import current_actor_context
-from leapp.models import (Report, SELinuxFacts, SelinuxPermissiveDecision,
-                          SelinuxRelabelDecision)
 
 
 # FIXME: fix the file properly regarding the fix of the issue:
```

Comment:
The mentioned issue is actually closed, is this comment still needed? https://github.com/oamg/leapp-repository/issues/20